### PR TITLE
Android code generation using Mustache templates

### PR DESCRIPTION
**Pull request by S2MBO**

### CODE REVIEW

File: metajson/readJSON.py [hunk 1]
```diff
@@ -198,10 +160,10 @@ def usage():
     usageString += '  -h, --help            shows help\n'
     usageString += '  -p, --prefix=         project prefix (default: S2M)\n'
     usageString += '  -s, --suffix=         classname suffix (default: JSONObject). Use "-s false" for no suffix\n'
-    usageString += '  -t, --target=         target platform iOS or Android (default: iOS)\n'
     usageString += '  -i, --input=          metaJSON file to read\n'
     usageString += '  -o, --output=         output path of generated source codes (default: src)\n'
     usageString += '      --template=       template directory to use to generate code\n'
+    usageString += '      --package=        name of the generated package (default: none)\n'
```
Comment (netbe) [inline metajson/readJSON.py:166]: Don't get what's package name, except it is specific to Java, right?

Comment (S2MBO) [inline metajson/readJSON.py:166]: Yes, it's important for Java but may also apply to other languages like Python. For Example: https://github.com/S2MBO/metaJSON/blob/feature/refactor/metajson/templates/Android/_CLASSNAME_.java.mustache#L8

**Comment by netbe** [comment]
@S2MBO find someone with Android knowledge to review your template, maybe @ben-efiz ?

**Comment by ben-efiz** [comment]
When executing metaJSON with the following code:

python metajson-runner.py -p S2M -o com/example/project/model/ --template=metajson/templates/Android/ --package=com.example.project.model -i samples/product.json

it uses the default suffix "JSONObject". The files are called correspondingly but the class names don't contain "JSONObject" at the end.

Calling it with suffix parameter, e.g. -s false it works and the java code looks ok. Some improvements would one more line break after each line and putting attributes and methods in the correct order (attributes first, then methods) but i can imagine that is not that easy during parsing.

**Comment by S2MBO** [comment]
@ben-efiz I fixed the suffix issue and implemented your suggested improvements. Please reconsider.

**Pull request merged**